Gateway/Validator/CheckoutResponseValidator.php

```diff
@@ -69,25 +69,27 @@ public function validate(array $validationSubject)
         // validate result
         if (!empty($response['resultCode'])) {
             $payment->setAdditionalInformation('resultCode', $response['resultCode']);
+
```

Comment:
Could we outsourse those lines so the PaymentResponseHandler can benefit? These fields validation is common for both classes

Reply:
wanted to do it in a separate PR because this PR is already quite big.